src/main/resources/application.yml

```diff
@@ -28,7 +28,7 @@ spring:
             client-id: ${system.client.id}
             client-secret: ${system.client.secret}
             authorization-grant-type: client_credentials
-            scope: read
+            scope: read,write
```

Comment:
Needed to add write scope here - otherwise the POST to prison-api failed with a 403 Forbidden